src/ZipkinOpenTracing/Span.php

```diff
@@ -115,4 +129,14 @@ public function getBaggageItem($key)
     {
         return $this->context->getBaggageItem($key);
     }
+
+    public function setScope(Scope $scope)
```

Comment:
I'd rather pass the scope as the last argument in the constructor and `nil` for non active spans. What do you think?